### PR TITLE
wgo: 0.5.7 (new formula)

### PR DESCRIPTION
**Pull request by mfinelli**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/w/wgo.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  homepage "https://github.com/bokwoon95/wgo"
+  url "https://github.com/bokwoon95/wgo/archive/refs/tags/v0.5.6d.tar.gz"
+  sha256 "2c2cdfa06678fe58f6d402d6c088a7b4132a717239b74256dbccf1f3421ae7db"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    bin.mkpath
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "-o", bin, "./..."
```
Comment (chenrui333) [inline Formula/w/wgo.rb:12]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
```

File: Formula/w/wgo.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  desc "Watch arbitrary files and respond with arbitrary commands"
+  homepage "https://github.com/bokwoon95/wgo"
+  url "https://github.com/bokwoon95/wgo/archive/refs/tags/v0.5.6d.tar.gz"
+  sha256 "2c2cdfa06678fe58f6d402d6c088a7b4132a717239b74256dbccf1f3421ae7db"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    bin.mkpath
```
Comment (chenrui333) [inline Formula/w/wgo.rb:11]: you dont need this?
```suggestion
```

Comment (mfinelli) [reply Formula/w/wgo.rb:11]: I based this on the `age` formula, so I just took it from there. I'll remove it.

Comment (chenrui333) [reply Formula/w/wgo.rb]: oh, i see

**Comment by chenrui333** [comment]
except the upstream release scheme, others lgtm, thanks!!

**Comment by mfinelli** [comment]
great do you need anything else from me or are we just waiting for a response from upstream?

**Comment by chenrui333** [comment]
yeah, I would hope to update the formula to use a more semantic version so that the following releases can be autobumped.

**Comment by mfinelli** [comment]
I see that upstream released 0.5.7, so I updated this to use that instead

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@mfinelli, thanks for your contribution to Homebrew-core! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**